### PR TITLE
Update custom helper docs

### PR DESCRIPTION
**Pull request by PeterNgTr**
## Motivation/Description of the PR

Applicable helpers:

- [ ] Webdriver
- [ ] Puppeteer
- [ ] Nightmare
- [ ] REST
- [ ] Appium
- [ ] Protractor
- [ ] TestCafe

- Description of this PR, which problem it solves
- A link to the corresponding issue (if applicable).

## Type of change

- [ ] Breaking changes
- [ ] New functionality
- [ ] Bug fix
- [ ] Documentation changes/updates
- [ ] Hot fix
- [x] Markdown files fix - not related to source code

## Checklist:

- [ ] Tests have been added
- [ ] Documentation has been added (Run `npm run docs`)
- [ ] Lint checking (Run `npm run lint`)
- [ ] Local tests are passed (Run `npm test`)

### CODE REVIEW

File: docs/pageobjects.md [hunk 1]
```diff
@@ -185,8 +185,8 @@ class AttachFile {
 }
 
 // For inheritance
-module.exports = new AttachFile()
-module.exports.AttachFile = AttachFile
+exports.AttachFile = AttachFile
```
Comment (Vorobeyko) [inline docs/pageobjects.md]: Why you change it? 
I changed it in https://github.com/Codeception/CodeceptJS/pull/2086 because this export not working.

Comment (Vorobeyko) [reply docs/pageobjects.md]: Maybe you must make rebase by master?

Comment (PeterNgTr) [reply docs/pageobjects.md]: sorry, my bad. Didn't rebase with master.

File: docs/helpers.md [hunk 2]
```diff
@@ -232,7 +234,7 @@ class JSFailure extends codecept_helper {
   }
 
   _afterStep() {
-    if (this.err) throw new Error('Browser JS error '+this.err);
+    if (this.err) throw new Error('Browser JS error '+ this.err);
```
Comment (Vorobeyko) [inline docs/helpers.md]: why is the space after the plus and not before?

File: docs/pageobjects.md [hunk 3]
```diff
@@ -185,7 +185,7 @@ class AttachFile {
 }
 
 // For inheritance
-module.exports = new AttachFile()
+module.exports = new AttachFile();	
 module.exports.AttachFile = AttachFile
```
Comment (Vorobeyko) [inline docs/pageobjects.md]: Why do-comma not places here like prev line?

File: docs/helpers.md (hunk 2 above)
Comment (PeterNgTr) [reply docs/helpers.md]: it is just the format for better view, space is not appended to the string itself.

File: docs/pageobjects.md (hunk 3 above)
Comment (PeterNgTr) [reply docs/pageobjects.md]: The current file on master didn't have it on both line. I'll add them to both lines.
https://github.com/Codeception/CodeceptJS/commit/634801483de24179ce69312ca58d2cc4b4785b83

File: docs/helpers.md (hunk 2 above)
Comment (Vorobeyko) [reply docs/helpers.md]: :smile: Maybe add the space to before plus?

Comment (PeterNgTr) [reply docs/helpers.md]: @Vorobeyko I think using literal string will be better. :)

File: docs/helpers.md [hunk 4]
```diff
@@ -110,7 +112,7 @@ module.exports = MyHelper;
 
 ## Puppeteer Example
 
-Puppteer has [nice and elegant API](https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md) which you can use inside helpers. Access `page` instance via `this.helpers.Puppeteer.page` from inside a helper.
+Puppteer has [nice and elegant API](https://github.com/GoogleChrome/puppeteer/blob/master/docs/api.md) which you can be used inside helpers. Access `page` instance via `this.helpers.Puppeteer.page` from inside a helper.
```
Comment (elukoyanov) [inline docs/helpers.md]: puppEteer

Comment (elukoyanov) [inline docs/helpers.md]: `which you can be used inside helpers` ?
What the?
maybe `which can be used` or `which you can use`

**Pull request merged**